packages/nx-plugin/src/executors/cli/executor.int.test.ts

```diff
@@ -27,6 +27,11 @@ describe('runAutorunExecutor', () => {
   });
 
   it('should normalize context, parse CLI options and execute command', async () => {
+    expect(process.env).toStrictEqual(
+      expect.not.objectContaining({
+        CP_VERBOSE: 'true',
+      }),
+    );
```

Comment:
The [`toHaveProperty`](https://vitest.dev/api/expect.html#tohaveproperty) matcher would be a good fit here:

```suggestion
    expect(process.env).not.toHaveProperty('CP_VERBOSE', 'true');
```